source/crud/crud.rst

```diff
@@ -1810,9 +1815,14 @@ Q: Where is ``useCursor`` in AggregateOptions?
 Q: Where is ``singleBatch`` in FindOptions?
   Drivers have historically allowed users to request a single batch of results (after which the cursor is closed) by specifying a negative value for the ``limit`` option. For servers < 3.2, a single batch may be requested by specifying a negative value in the ``numberToReturn`` wire protocol field. For servers >= 3.2, the ``find`` command defines ``limit`` as a non-negative integer option but introduces a ``singleBatch`` boolean option. Rather than introduce a ``singleBatch`` option to FindOptions, the spec preserves the existing API for ``limit`` and instructs drivers to convert negative values accordingly for servers >= 3.2.
 
+Q: Why are client-side errors raised for some unsupported options?
```

Comment:
Good thinking adding this rationale. Maybe it is also worth mentioning that unacknowledged writes using OP_QUERY commands really would give the client acknowledgement of error, hence why they do not require a client side error.

Reply:
Good idea. Updated.